yang-packages/draft-ietf-netmod-yang-packages.xml

```diff
@@ -2135,8 +2137,14 @@ module ietf-yang-inst-data-pkg {
 {
   "ietf-yang-instance-data:instance-data-set": {
     "name": "example-ietf-network-device-pkg",
-    "pkg-schema": {
-       package: "ietf-yang-package-defn-pkg@0.1.0.json"
+    "content-schema": {
+      "pkg-schema": {
+         "name": "ietf-yang-package-defn-pkg",
+         "version": "0.1.0"
+         // TODO: ietf-yang-package-defn-pkg isn't defined
+         // Or do we just want to use the module form of
+         // content-schema like example-ietf-routing-pkg ?
+      }
     },
```

Comment:
Change the examples to all use pkg-schema.
Raise another issue about how to "register" this defn-pkg.